src/Sentry/SentryClient.cs

```diff
@@ -225,6 +241,24 @@ public void CaptureTransaction(Transaction transaction, Scope? scope, Hint? hint
         return transaction;
     }
 
+    /// <summary>
+    /// Captures one or more metrics to be sent to Sentry.
+    /// </summary>
+    internal void CaptureMetrics(IEnumerable<Metric> metrics)
+    {
+        _options.LogDebug("Capturing metrics.");
+        CaptureEnvelope(Envelope.FromMetrics(metrics));
+    }
+
+    /// <summary>
+    /// Captures one or more <see cref="CodeLocations"/> to be sent to Sentry.
+    /// </summary>
+    internal void CaptureCodeLocations(CodeLocations codeLocations)
+    {
+        _options.LogDebug($"Capturing code locations for period: {codeLocations.Timestamp}");
+        CaptureEnvelope(Envelope.FromCodeLocations(codeLocations));
```

Comment:
Are code locations not coupled with metrics? In which case they should be on the same envelope ideally

Reply:
I think I did that because of this:
https://github.com/getsentry/sentry-python/blob/67c963d9c8d5e7e9de6347aee0edcf0c58d9fb24/sentry_sdk/metrics.py/#L580-L581